directed_forgetting/experiment.js

```diff
@@ -66,6 +66,48 @@ var getInstructFeedback = function() {
 		'</p></div>'
 }
 
+var getTestFeedback = function() {
+	var global_trial = jsPsych.progress().current_trial_global
+	var data_length = 144
+	var start_cut = global_trial - data_length
+	
+	//below are counters to see if the subject is treating this task as a directed remembering as opposed to a directed forgetting task
+	var neg_respond_remember_total = 0
```

Comment:
You track pos and neg responses/remembers separately, but never use the separate variables. Just replace these with "remember_total" and "total_responses" and increment each one